src/intrinsics.cpp

```diff
@@ -1242,11 +1242,11 @@ static Value *emit_untyped_intrinsic(intrinsic f, Value *x, Value *y, Value *z,
     case check_top_bit:
         // raise InexactError if argument's top bit is set
         x = JL_INT(x);
-        raise_exception_if(builder.
-                           CreateTrunc(builder.
-                                       CreateLShr(x, ConstantInt::get(t, t->getPrimitiveSizeInBits()-1)),
-                                       T_int1),
-                           literal_pointer_val(jl_inexact_exception), ctx);
+        emit_inexacterror_if(builder.
+                             CreateTrunc(builder.
+                                         CreateLShr(x, ConstantInt::get(t, t->getPrimitiveSizeInBits()-1)),
+                                         T_int1),
+                             xtyp, jl_false, ctx);
```

Comment:
I just plucked `jl_false` out of thin air, desperate for an object with "type" `jl_value_t*`.